test/elm/interval/test.coffee

```diff
@@ -1660,18 +1659,9 @@ describe 'DateIntervalExpand', ->
     a = @nullBoth.exec(@ctx)
     should.not.exist(a)
 
-  it 'returns null when per is more precise than the interval ends', ->
+  it 'returns more precise intervals when per is more precise than the interval ends', ->
```

Comment:
The description of this test is off.  I think it should be "returns empty list when..."